Code/MinimalLib/common.h

```diff
@@ -890,10 +891,17 @@ std::string generate_aligned_coords(ROMol &mol, const ROMol &templateMol,
     if (allowRGroups) {
       molHs.reset(MolOps::addHs(mol));
       prbMol = molHs.get();
+      auto queryParams = RDKit::MolOps::AdjustQueryParameters::noAdjustments();
```

Comment:
To think about for the future: it's worth figuring out whether or not we can get rid of some of the duplicate chemical logic that's present in this file

Reply:
Yeah, I have had this in the back of my mind for a while now, your comment was the final push to tip it over the activation barrier.
I have just submitted #6368 to fix that.